Makefile

```diff
@@ -210,53 +211,67 @@ test.integration: test.integration.dbless test.integration.postgres
 .PHONY: test.integration.enterprise
 test.integration.enterprise: test.integration.enterprise.postgres
 
-.PHONY: test.unit
-test.unit:
-	@go test -v -race \
+.PHONY: _test.unit
+_test.unit:
+	go test -v -race $(GOFLAGS) \
 		-covermode=atomic \
 		-coverpkg=$(PKG_LIST) \
 		-coverprofile=coverage.unit.out \
 		./internal/... \
-		./pkg/...
+		./pkg/... 2>/dev/null
 
-.PHONY: test.integration.dbless
-test.integration.dbless:
+.PHONY: test.unit
+test.unit:
+	@$(MAKE) _test.unit GOFLAGS="$(GOFLAGS)"
+
+.PHONY: test.unit.pretty
+test.unit.pretty:
+	@$(MAKE) _test.unit GOFLAGS="-json" 2>/dev/null | $(GOTESTFMT_CMD)
+
+.PHONY: _check.container.environment
+_check.container.environment:
 	@./scripts/check-container-environment.sh
-	@TEST_DATABASE_MODE="off" GOFLAGS="-tags=integration_tests" KONG_CONTROLLER_FEATURE_GATES=$(KONG_CONTROLLER_FEATURE_GATES) \
-		go test -v -race \
+
+.PHONY: _test.integration
+_test.integration: _check.container.environment
+	TEST_DATABASE_MODE="$(DBMODE)" GOFLAGS="-tags=integration_tests" KONG_CONTROLLER_FEATURE_GATES=$(KONG_CONTROLLER_FEATURE_GATES) \
+		go test -v $(GOFLAGS) \
 		-timeout $(INTEGRATION_TEST_TIMEOUT) \
 		-parallel $(NCPU) \
 		-race \
 		-covermode=atomic \
 		-coverpkg=$(PKG_LIST) \
-		-coverprofile=coverage.dbless.out \
+		-coverprofile=$(COVERAGE_OUT) \
 		./test/integration
 
+.PHONY: test.integration.dbless
+test.integration.dbless:
+	@$(MAKE) _test.integration DBMODE=off GOFLAGS="-race" COVERAGE_OUT=coverage.dbless.out
```

Comment:
`-race` flag is already added in the `_test.integration` target, we do not need it here.

Reply:
Changed